internal/driver/glfw/menu_bar.go

```diff
@@ -199,6 +160,7 @@ func (r *menuBarRenderer) Refresh() {
 	r.Layout(r.b.Size())
 	r.background.FillColor = theme.ButtonColor()
 	r.background.Refresh()
+	r.ShadowingRenderer.RefreshShadow()
```

Comment:
FYI: I guess, this is not really related to the base widget change and therefore should have been in a separate commit, right?

Reply:
I had assumed it was something that I broke in the refactoring. If it was already broken then yes it should be a separate commit.
I don't think it's worth unpicking and force pushing a better version, but I will try to keep an eye out in future commits.
If you disagree say so and I'll rework the history